plugins/caches.js

```diff
@@ -332,6 +333,96 @@ exports.plugin = {
                     }
                 }
             }
+        }, {
+            method: 'DELETE',
+            path: '/caches/{scope}/{id}',
+            handler: async (request, h) => {
+                if (strategyConfig.plugin !== 's3') {
+                    return h.response();
+                }
+
+                let cachePath;
+                const apiUrl = config.get('ecosystem.api');
+                const payload = {
+                    url: `${apiUrl}/v4/isAdmin`,
+                    method: 'GET',
+                    headers: {
+                        Authorization: `Bearer ${request.auth.token}`,
+                        'Content-Type': 'application/json'
+                    },
+                    json: true
+                };
+
+                switch (request.params.scope) {
+                case 'events': {
+                    return h.response();
+                }
+                case 'jobs': {
+                    const jobIdParam = request.params.id;
+
+                    payload.qs = {
+                        jobId: jobIdParam
+                    };
+
+                    cachePath = `jobs/${jobIdParam}/`;
+                    break;
+                }
+                case 'pipelines': {
+                    const pipelineIdParam = request.params.id;
+
+                    payload.qs = {
+                        pipelineId: pipelineIdParam
+                    };
+
+                    cachePath = `pipelines/${pipelineIdParam}`;
+                    break;
+                }
+                default:
+                    return boom.forbidden('Invalid scope');
+                }
+
+                try {
+                    await req(payload, (err, response) => {
+                        if (!err && response.statusCode === 200) {
```

Comment:
the endpoint returns `true` or `false`. Let's check that?

Reply:
fixed